### PR TITLE
Changing logo partner to Logotron

### PR DESCRIPTION
**Pull request by donlair**
#### Changes proposed in this Pull Request

We're testing a new logo maker partnership with Logotron to see if our users enjoy that service more. Context available at pbBQWj-II-p2. The partnership is featured in the Quick Links section of My Home and on a card on Marketing Tools. This PR updates those areas.

The link is updated in D58673-code.

Screenshot of the My Home update
Screen Shot 2021-03-15 at 10 30 42 AM

Screenshot of the updated card on Marketing Tools:
Screen Shot 2021-03-15 at 10 33 46 AM

#### Testing instructions
* Patch your sandbox with D58673-code
* Update /etc/hosts to point wp.me to your sandbox
* Checkout this PR and start Calypso
* Navigate to My Home, confirm the quick link says `Logotron` and that the link goes to the Logotron+WPcom partnership page.
* navigate to `marketing/tools/SITESLUG` and confirm that the card now mentions Logotron and that the link goes to the partnership page.

Related to #
* D58673-code

### CODE REVIEW

File: client/my-sites/marketing/tools/index.tsx [hunk 1]
```diff
@@ -119,9 +119,9 @@ export const MarketingTools: FunctionComponent = () => {
 				<MarketingToolsFeature
 					title={ translate( 'Want to build a great brand? Start with a great logo' ) }
 					description={ translate(
-						'A custom logo helps your brand pop and makes your site memorable. Make a professional logo in a few clicks with our partner Fiverr.'
+						'A custom logo helps your brand pop and makes your site memorable. Make a professional logo in a few clicks with our partner Logotron.'
```
Comment (a8ci18n) [inline client/my-sites/marketing/tools/index.tsx:122]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5581773

Thank you @donlair for including a screenshot in the description! This is really helpful for our translators.

**Comment by travisw** [comment]
It works 👍 

Note the translation and redirect issues noted in D58673-code.

**Review by travisw — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5581773

Thank you @donlair for including a screenshot in the description! This is really helpful for our translators.

**Comment by dlind1** [comment]
Just FYI, removing the String Freeze label before the PR is merged means that the [Deliverable](https://translate.wordpress.com/deliverables/overview/5581773) in GlotPress would have all strings removed and they won't be translated until after the PR is merged (which also incorrectly shows the 'Translation has been finished' message). Was that intentional or should the strings be translated before the merge?

**Comment by donlair** [comment]
> Just FYI, removing the String Freeze label before the PR is merged means that the [Deliverable](https://translate.wordpress.com/deliverables/overview/5581773) in GlotPress would have all strings removed and they won't be translated until after the PR is merged (which also incorrectly shows the 'Translation has been finished' message). Was that intentional or should the strings be translated before the merge?

Given the timing requirements for this partnership, we're merging before the translations are done (I'm about to merge it now).

**Comment by dlind1** [comment]
> Given the timing requirements for this partnership, we're merging before the translations are done (I'm about to merge it now).

Sounds good 🙂 Btw, in case of changes with tight deadlines we could also use the Deliverables functionality to provide translations quickly after the PR is merged, without going through the normal process, which we successfully utilized before for the sales banners.

**Comment by donlair** [comment]
> > Given the timing requirements for this partnership, we're merging before the translations are done (I'm about to merge it now).
> 
> Sounds good 🙂 Btw, in case of changes with tight deadlines we could also use the Deliverables functionality to provide translations quickly after the PR is merged, without going through the normal process, which we successfully utilized before for the sales banners.

Oh, I didn't know that was an option! Is that process documented somewhere? It sounds super useful - are there any extra costs or other downsides with it?

**Comment by dlind1** [comment]
We have various posts on our P2 (search for `deliverables`) with the initial post for the feature available at pxLjZ-5Jb-p2. There shouldn't be any extra costs or real downsides and it's just a process we've implemented to make translations more continuous and utilize our internal resources for translations. It's currently only working for merged Calypso PRs and PRs with the String Freeze label (the main use case), as well as manually created deliverables, so there are some limitations.

**Comment by dlind1** [comment]
@donalirl I've added all appropriate strings related to the logo partner name to the deliverable and it's currently translated to 7 languages. You can check the status of the translations here: https://translate.wordpress.com/deliverables/overview/5581773

**Comment by donalirl** [comment]
> @donalirl I've added all appropriate strings related to the logo partner name to the deliverable and it's currently translated to 7 languages. You can check the status of the translations here: https://translate.wordpress.com/deliverables/overview/5581773

I think this was meant for @donlair :)

**Comment by dlind1** [comment]
@donalirl Indeed it was 😅

**Pull request merged**